### ISSUE TITLE
L<number> syntax from example is not working

### ISSUE THREAD

**Issue by kvnkho** [taichino/croniter#174]
Hello!

Maybe I'm doing something wrong here but I was trying to use the cron expression '0 0 * * 5#3,L5' taken from the example under the [Usage section](https://github.com/taichino/croniter#usage) in the README but I couldn't get it to work.

This code snippet:

```python
from croniter import croniter
from datetime import datetime
base = datetime(2010, 1, 25, 4, 46)
iter = croniter('0 0 * * 5#3,L5', base)
```

throws: `croniter.croniter.CroniterNotAlphaError: [0 0 * * 5#3,L5] is not acceptable`

Similarly, this returns False.
```python
from croniter import croniter
print(croniter.is_valid("0 0 * * 5#3,L5"))
```

I read that this is version 0.3.5 and up. I am seeing this behavior on version 0.3.37.

Thanks!

**Comment by kiorky** [comment]
please upgrade to latest croniter (>=1.0.15)... 0.3 is really, really old.

**Comment by kiorky** [comment]
(and ``L`` works as expected)

```doctest
>>> base = datetime(2010, 1, 25, 4, 46)
>>> iter = croniter('0 0 * * 5#3,L5', base)
>>> iter.get_next()
1264723200.0
```

**Comment by kiorky** [comment]
``L`` support for each field began to be introduced from ``1.0.8``

**Comment by kiorky** [comment]
(and we remade fixes on ``1.0.11``).

**Comment by kvnkho** [comment]
Ah I see. Thanks for the response!
